simTest/BrainSwap_test.go

```diff
@@ -9,8 +9,13 @@ import (
 	. "github.com/FactomProject/factomd/testHelper"
 )
 
-// Test brainswapping a follower and a leader and swap a follower and an audit at the same height in the same build
-func TestBrainSwap1(t *testing.T) {
+/*
+Test brainswapping F <-> L  and F <-> A
+
+follower and a leader + follower and an audit
+at the same height in the same build
+*/
+func TestBrainSwap(t *testing.T) {
```

Comment:
Request from Clay is to add a test that will do the brainswap some multiple number of times (10 to start)

If possible test calling checkprocesslist to log for debug + test for gaps